ffc-ahwr-mi-reporting/storage/storage.js

```
@@ -63,9 +63,16 @@ const processEntitiesByTimestampPaged = async (tableName, fileName, context) =>
     let rowContent = ''
 
     for await (const event of eventsPage) {
-      const csvRow = transformEventToCsvV3(event, context)
-      rowContent += csvRow + '\n'
-      eventItemCount++
+      try {
+        const csvRow = transformEventToCsvV3(event, context)
+        rowContent += csvRow + '\n'
+        eventItemCount++
+      } catch (err) {
+        context.log.error('Failed to transform event to csv.', {
```

Does this need to rethrow the error?

---

I think the plan is to allow individual events to fail and to just be logged so the remaining events can still be processed. I was thinking whether we should be throwing an error at teh end if there are any errors but in the jira it just says to throw an error if there is a catastrophic error

---

Yeah fair enough, makes sense to me